include/irr/config/BuildConfigOptions.h.in

```diff
@@ -15,6 +15,12 @@
 #cmakedefine _IRR_COMPILE_WITH_OPENGL_
 #cmakedefine _IRR_COMPILE_WITH_VULKAN_
 
+// loaders
+#cmakedefine _IRR_COMPILE_WITH_OPENEXR_LOADER_
+
+// writers
+#cmakedefine _IRR_COMPILE_WITH_OPENEXR_WRITER_
+
```

Comment:
@AnastaZIuk can you do this for all loaders (and obviously pick good defaults)

Reply:
sure!

Reply:
after PR merging of course, before I handle GLI